### ISSUE TITLE
Android only: the ui-material-tabs version 6.2.20 breaks page navigation in Angular app

### ISSUE THREAD

**Issue by cjohn001** [nativescript-community/ui-material-components#365]
When navigation away from a page with mdtabs and than trying to navigate back or to a further page, navigation does not work anymore. 

No error messages are shown.

@nativescript-community/ui-material-tabs
### Which platform(s) does your issue occur on?
- android 
- Android version API 31
- only emulator tested 

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.1.5-next-03-03-2022-1927554528
- Cross-platform modules: @nativescript/core": "8.1.5"
- Runtime(s): "@nativescript/android": "8.2.0-alpha.11"
- Plugin(s): 
```
	"dependencies": {
		"@angular/animations": "13.2.5",
		"@angular/common": "13.2.5",
		"@angular/compiler": "13.2.5",
		"@angular/core": "13.2.5",
		"@angular/forms": "13.2.5",
		"@angular/platform-browser": "13.2.5",
		"@angular/platform-browser-dynamic": "13.2.5",
		"@angular/router": "13.2.5",
		"@apollo/client": "3.5.10",
		"@nativescript-community/ui-material-tabs": "6.2.20",
		"@nativescript/angular": "13.0.3",
		"@nativescript/core": "8.1.5",
		"@nativescript/iqkeyboardmanager": "2.0.0",
		"@nativescript/localize": "5.0.4",
		"@nativescript/mlkit-barcode-scanning": "1.0.0-alpha.8",
		"@nativescript/mlkit-core": "1.0.0-alpha.8",
		"@nativescript/secure-storage": "3.0.0",
		"@nativescript/theme": "3.0.2",
		"@nativescript/ui-charts": "file:../mnd-custom-plugins/nativescript-ui-charts/publish/package/nativescript-ui-charts-0.1.10.tgz",
		"apollo-angular": "3.0.0",
		"apollo3-cache-persist": "0.14.0",
		"d3-ease": "3.0.1",
		"graphql": "16.3.0",
		"graphql-tag": "2.12.6",
		"intl": "1.2.5",
		"moment": "2.29.1",
		"nativescript-oauth2-ext": "file:../mnd-custom-plugins/nativescript-oauth2-ext/publish/package/nativescript-oauth2-ext-3.0.0.tgz",
		"nativescript-sqlite": "2.8.6",
		"nativescript-ui-autocomplete": "8.0.1",
		"nativescript-ui-calendar": "file:../mnd-custom-plugins/plugins-ui/dist/packages/ui-calendar/nativescript-ui-calendar-8.0.2.tgz",
		"nativescript-ui-gauge": "8.0.1",
		"rxjs": "7.5.4",
		"uuid": "8.3.2",
		"zone.js": "0.11.4"
	},
```

**Comment by cjohn001** [comment]
resolved with 6.2.23

**Comment by farfromrefug** [comment]
@cjohn001 thanks very good to hear!
